.github/workflows/go.yml

```
@@ -22,19 +22,23 @@ jobs:
       pull-requests: read # For golangci-lint annotations
     steps:
     - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)